client/client_state.h

```diff
@@ -62,130 +62,12 @@ using std::vector;
 #include "../sched/edf_sim.h"
 #endif
 
-//////// TIME-RELATED CONSTANTS ////////////
-
-#define POLL_INTERVAL   1.0
-    // the client will handle I/O (including GUI RPCs)
-    // for up to POLL_INTERVAL seconds before calling poll_slow_events()
-    // to call the polling functions
-
-#define GARBAGE_COLLECT_PERIOD  10
-    // how often to garbage collect
-
-#define TASK_POLL_PERIOD    1.0
-
-#define UPDATE_RESULTS_PERIOD   1.0
-
-#define HANDLE_FINISHED_APPS_PERIOD 1.0
-
-#define BENCHMARK_POLL_PERIOD   1.0
-
-#define PERS_FILE_XFER_START_PERIOD  1.0
-#define PERS_FILE_XFER_POLL_PERIOD  1.0
-
-#define SCHEDULER_RPC_POLL_PERIOD   5.0
-
-#define FILE_XFER_POLL_PERIOD   1.0
-
-#define GUI_HTTP_POLL_PERIOD    1.0
-
-#define MEMORY_USAGE_PERIOD     10
-    // computer memory usage and check for exclusive apps this often
-
-//////// WORK FETCH
-
-#define WORK_FETCH_PERIOD   60
-    // see if we need to fetch work at least this often
-#define WF_MIN_BACKOFF_INTERVAL    600
-#define WF_MAX_BACKOFF_INTERVAL    86400
-    // if we ask a project for work for a resource and don't get it,
-    // we do exponential backoff.
-    // This constant is an upper bound for this.
-    // E.g., if we need GPU work, we'll end up asking once a day,
-    // so if the project develops a GPU app,
-    // we'll find out about it within a day.
-
-#define WF_UPLOAD_DEFER_INTERVAL   300
-    // if a project is uploading,
-    // and the last upload started within this interval,
-    // don't fetch work from it.
-    // This allows the work fetch to be merged with the reporting of the
-    // jobs that are currently uploading.
-
-#define RESULT_REPORT_IF_AT_LEAST_N 64
-    // If a project has at least this many ready-to-report tasks, report them.
-
 #define WF_EST_FETCH_TIME 180
```

Comment:
I guess `WF_EST_FETCH_TIME` should go with the rest of the constants around line 570.

Reply:
@davidpanderson

Please move the `#define` and the comment following it. It got relocated in the merge commit.